src/amo/routes.js

```diff
@@ -30,7 +30,7 @@ import ServerError from './components/ErrorPage/ServerError';
 export default (
   <Route path="/:lang/:application" component={App}>
     <IndexRoute component={Home} />
-    <Route path="addon/:slug/" component={DetailPage} />
+    <Route path="addon/:slug/" component={AddonDetail} />
```

Comment:
I wonder if `AddonDetail` should just be `Addon`? I think that gets to the core of what that page is and it's very central to the site so such a bold name doesn't seem off to me. We don't have a generic `Addon` component to display bits of an `Addon` and I wonder if we ever will... or if we do we can go back to calling this one `AddonDetail`.

Just a thought, I don't care *that* much either way, but given there is no `Addon` or `AddonSnippet` or anything else like that, I wonder why we're specific with this component's name. It could just be `Addon`.